packages/ord-connect/src/components/SelectWalletModal/index.tsx

```diff
@@ -414,31 +414,31 @@ export function SelectWalletModal({
                         isMobileDevice={isMobile}
                         renderAvatar={renderAvatar}
                       />
-                      <hr className="horizontal-separator" />
                       {!isMobile && (
```

Comment:
currently `<WalletButton/>` is repeated for 4 times here, we can create a constant in which it is an array of object that consists of whether or not this wallet is available on mobile, e.g.: [{name: xverse, availableOnMobile: true}, {name: unisat, ...}, ... ] 

Then in the rendering logic we can make sure of `.map`

Reply:
I think `<hr />` can also be conditionally rendered based on the number of wallets that will be displayed

Reply:
Let's do the refactor another time, we'll remove the extra horizontal separators first